fdbclient/include/fdbclient/StatusClient.h

```diff
@@ -28,7 +28,7 @@
 class StatusClient {
 public:
 	enum StatusLevel { MINIMAL = 0, NORMAL = 1, DETAILED = 2, JSON = 3 };
-	static Future<StatusObject> statusFetcher(Database db);
+	static Future<StatusObject> statusFetcher(Database db, std::string statusField = "");
```

Comment:
Since the current implementation only supports a limited fields from the status json. It's better to document them here, so that we won't be confused in the future and assume any field can be specified.

Reply:
Yes, I will do that.

Reply:
Done.